desk_test.go

```diff
@@ -53,14 +53,15 @@ func (*testSettings) Background() string {
 
 func TestDeskLayout_Layout(t *testing.T) {
 	l := &deskLayout{}
-	l.background = canvas.NewRectangle(color.White)
+	l.screens = l
+	l.backgrounds = append(l.backgrounds, canvas.NewRectangle(color.White))
```

Comment:
l.backgrounds expects a slice of []fyne.CanvasObject - perhaps because I'm newish to Go I'm not sure how to assign an array of [1]fyne.CanvasObject to that?  I get errors.

Reply:
Yes I see that the layout contains a Screens type,  but I don’t understand why there is the additional implementation of the interface in the layout code here...

Reply:
[]fyne.CanvasObject{element1}